scripts/Deployment.s.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: MIT
```

Comment:
@0xMotto which method of verification will we use along with this deployment. Remember verifying is a pta on canto.